test/OrchardCore.Commerce.Tests.UI/Tests/OrderTests/BehaviorOrderTests.cs

```diff
@@ -141,14 +134,75 @@ public BehaviorOrderTests(ITestOutputHelper testOutputHelper)
                     .ShouldContain($"/Admin/Contents/ContentItems/{TestProduct}/Edit");
 
                 // Product should be deletable from list after it has been submitted.
-                await context.ClickReliablyOnAsync(By.XPath("//button[contains(@class, 'btn btn-danger')][1]"));
+                await ClickDeleteItemAsync(context, 1);
                 await context.ClickPublishAsync();
 
                 context.Missing(ByQuantity(1));
             },
             browser);
 
-    private static Task AddNewProductAsync(UITestContext context) => context.ClickReliablyOnAsync(By.Id("addButton"));
+    [Theory, Chrome]
+    public Task OrderPaymentRequestButtonShouldWorkProperly(Browser browser) =>
+        ExecuteTestAfterSetupAsync(
+            async context =>
+            {
+                await context.SignInDirectlyAndGoToDashboardAsync();
+                await context.CreateNewContentItemAsync(Order);
```

Comment:
Could you create a pending order as sample recipe content? It would be good for demoing as well as make testing here simpler.